### PR TITLE
Update cptx.cfg

### PR DESCRIPTION
**Pull request by sachendras**
Propose cPTX configuration as per [topology](https://docs.google.com/drawings/d/1ZT7ZlQ-yB_JYZhjLhppzpTkPnWz_NuL5zkpaWFymHw4/edit?resourcekey=0-dihvJFJ81_mubhN_Bih-CA)

XRD configuration updated in https://github.com/openconfig/kne/pull/358

### CODE REVIEW

File: examples/multivendor/cptx.cfg [hunk 1]
```diff
@@ -21,8 +21,257 @@ interfaces {
     re0:mgmt-0 {
         unit 0 {
             family inet {
-                address FXP0ADDR;
```
[the remark this answers is not in the document]
Comment (sachendras) [reply examples/multivendor/cptx.cfg:24]: You asking about re0:mgmt-0 or FXP0ADDR?  FXP0ADDR is just a variable which will be replaced by an IP address during boot.

[the remark this answers is not in the document]
Comment (nitinsoniism) [reply examples/multivendor/cptx.cfg:24]: FXP0ADDR. That's correct hence it should not be changed to re0:mgmt-0. Also, re1 part of the mgmt iface config will have no effect.

[the remark this answers is not in the document]
Comment (sachendras) [reply examples/multivendor/cptx.cfg:24]: ack, corrected

[the remark this answers is not in the document]
Comment (nitinsoniism) [reply examples/multivendor/cptx.cfg:24]: My bad, I meant the address change and not the mgmt iface name change. Address should remain as FXP0ADDR.

**Comment by alexmasi** [comment]
The presubmit failure is due to a cptx timeout for generating certs. This timeout value was increased in https://github.com/openconfig/kne/pull/364 which is not included in this PR. However this change affects nothing tested by the presubmit as its just an example config so it is safe to merge even with presubmit failure.

**Pull request merged**